massa-client/src/cmds.rs

```diff
@@ -251,6 +261,30 @@ pub enum Command {
     when_moon,
 }
 
+#[derive(Debug, Display, EnumString, EnumIter)]
+#[strum(serialize_all = "snake_case")]
+pub enum CLIOperation {
```

Comment:
Let's rename this to `ListOperations` since those are the only commands that make use of that

Reply:
Checked and it's fixed. Closing it...